src/docker-compose.override.yml

```diff
@@ -17,3 +17,6 @@ services:
       - EVENTGRID_AUTH_KEY=<Event Grid Auth Key>
     ports:
       - '42064:5001'
+  arcus.messaging.tests.runtimes.azurefunction.servicebus.queue:
```

Comment:
We don't need this since Docker Compose doesn't like Azure Functions

https://github.com/microsoft/DockerTools/issues/124